### ISSUE TITLE
Jitsi's window is rendered twice when it loses focus

### ISSUE THREAD

**Issue by WhyNotHugo** [jitsi/jitsi-meet-electron#623]
### Description:

When another window is focused, Jitsi's window get duplicated.

A video of the issue (the window on the left it Jitsi, the one on the middle/right is an alacritty terminal).

https://user-images.githubusercontent.com/730811/129249475-c6dc547a-f260-412e-95d2-7274bf7da2c8.mp4

If I'm sharing screen or webcam, the duplicate renders this too.

It also steals focus from whatever other window had focus, and the only way to focus a non-Jitsi window is to focus on that duplicate and then back to another window.

I'm using the flatpak version, on sway, but it's running via XWayland (e.g.: using its xorg backend).

### Steps to reproduce:

1. Join any conference
2. Focus another window

### Expected behavior:

Nothing crazy should happen.

### Actual behavior:

Jitsi's window get duplicated, and the duplicate steals focus.

### Server information:

Using the default / hosted version.

### Client information:

- Browser / app version: app 2.8.10, install via Flatpak
- Operating System: ArchLInux

### Additional information:

In case this is relevant, I'm running wayland, but Jitsi itself is running via XWayland, so is itself using its xorg backend.

I tried using `org.jitsi.jitsi-meet --enable-features=UseOzonePlatform --ozone-platform=wayland` to see if it can use Electron's native support, but that doesn't work so I'll leave that as out-of-scope here.

**Comment by saghul** [comment]
That's the "always on top" window. It's a special window which shows above other windows. This of course doesn't make sense when using a tiling window manager. You can disable this from the settings menu, before you join any meeting.

**Comment by WhyNotHugo** [comment]
Oh, thanks. Does this window normally cover the whole desktop so as to work in a "kiosk" sort of form?

If so, wouldn't it make more sense to fullscreen the regular window?

**Comment by saghul** [comment]
No, it's show above any other window, in a small size. Its purpose is to allow users to do other tasks while still having a small view of the conference.

**Comment by WhyNotHugo** [comment]
Ah, this was in the [README](https://github.com/jitsi/jitsi-meet-electron-utils/blob/master/README.md#always-on-top) :facepalm:  (it took me a while to find that though).

Actually, windows can be marked as a floating window so tiling WMs will treat it as such.

**Comment by saghul** [comment]
> Actually, windows can be marked as a floating window so tiling WMs will treat it as such.

Good to know!

**Comment by WhyNotHugo** [comment]
Anyway, thanks for the help, turning that off worked for me.
